### DISCUSSION TITLE
Is it possible to put the ComboBox label on the left side?

### DISCUSSION THREAD

**Discussion by vbeffa** [emilk/egui#925] [Q&A]
Is it possible to put the ComboBox label on the left side?

**Comment by emilk** [comment]
Nope. Make a PR!

**Comment by gilescope** [comment]
empty string and a label before it works as a workaround.

**Comment by ThatOneShortGuy** [comment]
For those in the future implementing the workaround suggested by @gilescope, don't do an empty string as it may cause an ID collision. Use `ComboBox::from_id_source` instead. I spent too long trying to figure out where the collision was smh.
